src/main.rs

```diff
@@ -287,12 +286,24 @@ mod app {
             };
 
             // Handle the MQTT control interface.
-            let main_bus = &mut c.shared.main_bus;
             c.shared
                 .net_devices
                 .lock(|net| {
-                    match net.control.poll(|handler, topic, data, output| {
-                        main_bus.lock(|bus| handler(bus, topic, data, output))
+                    match net.control.poll(|command, data, output| match command {
+                        "read-bias" => c
+                            .shared
+                            .main_bus
+                            .lock(|bus| crate::net::mqtt_control::read_bias(bus, data, output)),
+                        "save" => {
+                            (&mut c.shared.main_bus, &mut c.shared.usb_terminal).lock(|bus, usb| {
+                                crate::net::mqtt_control::save_settings_to_flash(
+                                    bus,
+                                    usb.platform_mut(),
+                                    data,
+                                )
+                            })
+                        }
+                        _ => unreachable!(),
```

Comment:
It would be really cool to define the minireq handlers like in https://github.com/rust-embedded-community/menu

Reply:
The main issue is that the function signature of a handler is inconsistent, so we can't defined compile-time storage for them anywhere.

Reply:
Same for the `menu` handlers. They get a context to call into.

Reply:
Yeah, but the problem with a Context is that it can't have anonymous lifetime borrows in it. We got around this by borrowing the context itself, but that doesn't work if you need _two_ lockable items